README.md

```diff
@@ -6,6 +6,37 @@ This repository is the [r10k](https://github.com/adrienthebo/r10k) control
 repository for the [Jenkins](https://jenkins.io) project's own
 infrastructure.
 
+## Structure
+
+See the [Jenkins infrastructure project](https://jenkins.io/projects/infrastructure/) for overview of the project's infrastructure and the services being managed by this repository.
+Full list of services is available [here](https://jenkins.io/projects/infrastructure/#services).
+
+### Implementation notes
+
+* The services are managed [r10k](https://github.com/adrienthebo/r10k) and Puppet,
+  configuration files are available inside this repository.
+* There are multiple types of service deployments:
+  * The majority of services run as containers inside Kubernetes, and it is a precondition for new services.
```

Comment:
I should lkely add a reference to https://github.com/jenkins-infra/charts , CC @halkeye

Reply:
```suggestion
  * The majority of services run as containers inside Kubernetes, and it is a precondition for new services.
    There is ongoing work on [Jenkins Infra Helm charts](https://github.com/jenkins-infra/charts) which are expected to replace puppet deployments in the future.
```